app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    render json: Greeting.find(rand(1..5))
```

Comment:
- [ ] There are no greetings being found on your database, kindly populate it 
![image](https://user-images.githubusercontent.com/62705947/225954258-dd6cc86a-94a7-4346-9717-db2782ac2f7a.png)